### ISSUE TITLE
Pomalá reakce na klikání v modalech

### ISSUE THREAD

**Issue by motyc** [ARUP-CAS/aiscr-webamcr#1136]
Aplikace má dlouhou prodlevu při posunech stavů a obecně při operacích s modal okny - po kliknutí na tlačítko se zdánlivě chvíli nic neděje a až po chvilce se vše rozběhne.

**Comment by motyc** [comment]
@jnihnat Ano, to by určitě pomohlo. Jde o to, že teď to může uživatele vést k potřebě zběsile klikat, protože to vypadá, že se nic neděje. Pokud to tedy není problém, prosím loader přidat.

**Comment by jnihnat** [comment]
Tak nakonec pridany i ten Loader k tlacitku po odeslani formulare. Pro jistotu prokontrolujte par modalu ze to funguje spravne.
